src/shared/Form/ui/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Container, Form } from "react-bootstrap"
+
+export type BasicFormProps = React.FormHTMLAttributes<HTMLFormElement>
```

Comment:
"You can use PropsWithChildren instead"
same here

Reply:
Done